doc/api/stream.md

```diff
@@ -2080,10 +2080,14 @@ added: v16.9.0
 * `streams` {Stream\[]|Iterable\[]|AsyncIterable\[]|Function\[]}
 * Returns: {stream.Duplex}
 
-Combines two or more streams into a `Duplex` stream that writes to the
-first stream and reads from the last. Each provided stream is piped into
-the next, using `stream.pipeline`. If any of the streams error then all
-are destroyed, including the outer `Duplex` stream.
+Combines two or more `Duplex` streams (e.g. transforms) into a `Duplex`
+stream that writes to the first stream and reads from the last. Each provided
+stream is piped into the next, using `stream.pipeline`. If any of the streams
+error then all are destroyed, including the outer `Duplex` stream.
+
+It should not be confused with `stream.Duplex.from({writable, readable})`
+which can combine a write-only stream with a read-only stream to create a
```

Comment:
Existing docs don't use the terms "write-only" and "read-only". In addition, we can generalize the comparison to `stream.Duplex.from()` because it's not necessarily about "write-only" or "read-only" as in _having_ only a writable or readable side. Rather, it's about whether to _use_ those sides.

So for this paragraph I suggest:

> Use other utilities when the streams are not `Duplex` or when connecting them is not desirable. For example, to instead create a `Duplex` stream from a writable stream and a readable stream without piping one into the other, use `stream.Duplex.from()`.

In addition, can we move it down by one paragraph? Because describing the difference between `stream.compose()` and `stream.pipeline()` is IMO more important than describing the difference between `stream.compose()` and `stream.Duplex.from()`. And the first paragraph already mentions `stream.pipeline()` so that question is brought up first.